sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/filter/Predicate.java

```diff
@@ -146,4 +149,210 @@ public class Predicate extends GeneralScalarExpression {
   public Predicate(String name, Expression[] children) {
     super(name, children);
   }
+
+  public org.apache.spark.sql.sources.Filter toV1() {
+    String expressionStr = "";
+    for (Expression e : children()) {
+      expressionStr += e.describe() + ", ";
+    }
+
+    if (name().equals("=")) {
+      if (children()[1] instanceof LiteralValue) {
+        // e.g. a = 1
+        return new EqualTo(children()[0].describe(),
+            CatalystTypeConverters.convertToScala(((LiteralValue)children()[1]).value(),
```

Comment:
Two indents

Reply:
I changed my ide java continuation indent to 4 for another project. Changed back to 2 space. All the indentation should be good now.